README.md

```diff
@@ -41,6 +41,7 @@ Databases on SQL Server, Azure SQL Database, or Azure SQL Managed Instance which
 
 ## Known Issues
 - The table used by a SQL binding or SQL trigger cannot contain two columns that only differ by casing (Ex. 'Name' and 'name'). 
+- Non-CSharp functions using SQL bindings against tables with columns of data types `BINARY` or `VARBINARY` need to map those columns to a string type. Input bindings will return the binary value as a base64 encoded string. Output bindings require the value upserted to binary columns to be a base64 encoded string.
```

Comment:
Shouldn't this be the below? If they turn it into a string themselves then it'll stay as a string, right?

```suggestion
- Non-CSharp functions using SQL bindings against tables with columns of data types `BINARY` or `VARBINARY` need to map those columns to a string type. If not then input bindings will return the binary value as a base64 encoded string and output bindings require the value upserted to binary columns to be a base64 encoded string.
```

Reply:
For input bindings, the binary values are sent to the function as a base64 string. For output bindings, the value being upserted needs to be converted to a base64 string.

If we try to use a byte array in a non-CSharp output binding, we get the following error because Newtonsoft expects a string:
System.Private.CoreLib: Exception while executing function: Functions.AddProductColumnTypes. Microsoft.Azure.WebJobs.Host: Error while handling parameter _binder after function returned:. Newtonsoft.Json: Error reading string. Unexpected token: StartArray. Path 'Binary', line 23, position 13.